source/aws-bootstrap-kit/lib/secure-root-user.ts

```diff
@@ -46,14 +51,89 @@ export class SecureRootUser extends core.Construct {
       }
     );
 
+    // Create role used for auto remediation
+    const autoRemediationRole = new iam.Role(this, 'AutoRemediationRole', {
+      assumedBy: new iam.CompositePrincipal(
+          new iam.ServicePrincipal("events.amazonaws.com"),
+          new iam.ServicePrincipal("ssm.amazonaws.com"),
+      )
+    });
+
+    autoRemediationRole.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AmazonSSMAutomationRole'));
```

Comment:
(Optional)
Note that this role allows `ec2:StartInstances` on resource `*`. But I think it is safe as we only grant principal SSM and events.

Reply:
Good point. Will avoid managed policy